lib/model/rwfolder.go

```diff
@@ -295,14 +298,7 @@ func (f *sendReceiveFolder) pull(prevIgnoreHash string) (curIgnoreHash string, s
 		}
 	}
 
-	if len(toBeScanned) != 0 {
-		scanList := make([]string, len(toBeScanned))
-		for path := range toBeScanned {
-			l.Debugln(f, "scheduling scan after pulling for", path)
-			scanList = append(scanList, path)
-		}
-		go f.Scan(scanList)
-	}
+	close(scanChan)
```

Comment:
This probably should be after the FolderIdle as you might bulldoze FolderScan state.

Reply:
It doesn't matter as the actual scan will happen via the main folder for loop, i.e. after this function finished. I'll change it to prevent that confusion regardless.